frontend-react/src/components/Spinner.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react'
+import { SpinnerCircular } from 'spinners-react'
+
+function Spinner() {
+    return (
+        <div id="spinner" className="grid-container">
+            <SpinnerCircular
+                color="rgba(57, 88, 172, 1)"
```

Comment:
Probably better to have color come out of the scss them via a `className`, but this component doesn't look like it works like that. Lots of other knobs we can turn at some point, too like `secondaryColor`, `speed`, `thickness`.
So, no change really required here.